.env.example

```
@@ -0,0 +1,30 @@
+# Stargate V2 Endpoints
+ENDPOINT_ETHEREUM=0x1a44076050125825900e736c501f859c50fE728c
+ENDPOINT_BASE=0x1a44076050125825900e736c501f859c50fE728c
+ENDPOINT_BNB=0x1a44076050125825900e736c501f859c50fE728c
```

```suggestion
STARGATE_ENDPOINT_SMARTCHAIN=0x1a44076050125825900e736c501f859c50fE728c
```

let's keep it consistent with our internal naming